conf/messages.de

```diff
@@ -32,6 +32,7 @@ securesocial.signup.thankYouCheckEmail=Vielen Dank. Bitte prüfen Sie ihr E-Mail
 securesocial.signup.invalidLink=Der Link ist ungültig
 securesocial.signup.signUpDone=Vielen Dank für Ihre Registrierung.
 securesocial.signup.invalidPassword=Das Passwort muss mindestens {0} Zeichen, eine Ziffer, einen Großbuchstaben und einen Kleinbuchstaben enthalten.
+securesocial.signup.passwordInfoOnChange=Das neue Passwort muss mindestens {0} Zeichen, eine Ziffer, einen Großbuchstaben und einen Kleinbuchstaben enthalten.
```

Comment:
Duplicate strings. Do we really need "neue Passwort" here? Couldn't both texts just be the same?